src/AddThreepid.ts

```diff
@@ -17,18 +17,19 @@ limitations under the License.
 */
```

Comment:
The remaining [strict TypeScript lints](https://github.com/matrix-org/matrix-react-sdk/actions/runs/4579683104/jobs/8087699269?pr=10432) are tracked by https://github.com/vector-im/element-web/issues/24899 and there is one unrelated lint on a piece of code that wasn't touched and would be tangent to these error string translations fixes.